deploy/yamls/ubuntu/zfs-driver.yaml

```diff
@@ -791,11 +808,42 @@ spec:
               mountPath: /lib/libuutil.so.1
             - name: libnvpair
               mountPath: /lib/libnvpair.so.1
+            - name: exportfs
+              mountPath: /usr/sbin/exportfs
+            - name: mount-nfs
+              mountPath: /usr/sbin/mount.nfs
+            - name: rpc-nfsd
+              mountPath: /usr/sbin/rpc.nfsd
+            - name: rpc-mountd
+              mountPath: /usr/sbin/rpc.mountd
+            - name: libtirp
+              mountPath: /usr/lib/x86_64-linux-gnu/libtirpc.so.3
             - name: pods-mount-dir
               mountPath: /var/lib/kubelet/
+              mountPropagation: "Bidirectional"
               # needed so that any mounts setup inside this container are
               # propagated back to the host machine.
+            - name: libgssapi
+              mountPath: /usr/lib/x86_64-linux-gnu/libgssapi_krb5.so.2
+            - name: libkrb
+              mountPath: /usr/lib/x86_64-linux-gnu/libkrb5.so.3
+            - name: libkrbcrypto
+              mountPath: /usr/lib/x86_64-linux-gnu/libk5crypto.so.3
+            - name: libkrbsupport
+              mountPath: /usr/lib/x86_64-linux-gnu/libkrb5support.so.0
+            - name: libkeyutils
+              mountPath: /usr/lib/x86_64-linux-gnu/libkeyutils.so.1
+            - name: libwrap
+              mountPath: /usr/lib/x86_64-linux-gnu/libwrap.so.0
+            - name: exports
+              mountPath: /etc/exports
+            - name: nfs-state
+              mountPath: /var/lib/nfs
+            - name: tank
```

Comment:
what is this directory?

Reply:
That is the "canonical" dataset root for any datasets under the zpool named tank